media-autobuild_suite.bat

```diff
@@ -591,6 +591,26 @@ if %buildfaac%==2 set "faac=n"
 if %buildfaac% GTR 2 GOTO faac
 if %deleteINI%==1 echo.faac=^%buildfaac%>>%ini%
 
+:exhale
+if %exhaleINI%==0 (
+    echo -------------------------------------------------------------------------------
+    echo -------------------------------------------------------------------------------
+    echo.
+    echo. Build exhale binary? [open-source ISO/IEC 23003-3 USAC, xHE-AAC encoder]
```

Comment:
Maybe you should remove the check for `$standalone = y` and mention here `Binaries being built do not depend on "standalone=y"`

Since standalone is technically tied to building binaries for libraries included in FFmpeg
```none
Build standalone binaries for libraries included in FFmpeg?
```

Also, can you check if the exhale.exe that is produced is static? run `ldd /local64/bin-audio/exhale.exe`

Reply:
Added the note in [this commit](https://github.com/m-ab-s/media-autobuild_suite/pull/1632/commits/a33f6683d0fced3593debf6eb90851f3f00a5aa3)